phoenix-pherf/src/main/java/org/apache/phoenix/pherf/result/QueryResult.java

```diff
@@ -150,6 +151,7 @@ private long getResultRowCount() {
                 List<ResultValue> rowValues = new ArrayList<>();
                 rowValues.add(new ResultValue(util.convertNull(getStartTimeText())));
                 rowValues.add(new ResultValue(util.convertNull(this.getQueryGroup())));
+                rowValues.add(new ResultValue(util.convertNull(this.getId())));
```

Comment:
This allows me to verify the results for each query, since all executions of query q1 are supposed to fail and all of the ones for q2 and q3 are supposed to succeed
Since my test queries are all in one scenario file now

Reply:
Ok, so the result file henceforth would also have these IDs in them right? Does this require a documentation change?

Reply:
It doesn't look like there's any documentation re: output files to begin with, at least not as far as I can see

Reply:
Let's add it to your documentation changes as part of PHOENIX-5818